http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/HttpServletWithSessionListenerIT.java

```diff
@@ -51,6 +57,30 @@ public void sessionSecured() {
         thenCheckActiveSessionsEqualTo(1);
     }
 
+    @Test
+    @DisplayName("undertow servlet test http2")
+    public void underTowHttp2Servert(VertxTestContext vertxTestContext) {
+        final String HTTP_2_VERSION = "HTTP_2";
+        Vertx vertx = Vertx.vertx();
+        String uriUnderTow = "/app/servlet/hello";
+        String expectedResponseFromUndertow = "Hello World";
+        HttpClientOptions options = new HttpClientOptions().setProtocolVersion(HttpVersion.HTTP_2);
+        HttpClient httpClient = vertx.createHttpClient(options);
```

Comment:
this could lead to resource leak, HttpClient is closeable

Reply:
Added 
```
httpClient.close();
vertx.close();
```